test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
Shouldn't that use `[ValidateAntiForgeryToken]`? In https://github.com/aspnet/MusicStore/commit/ca515e3e875b97e9327aefc1646fdca018e12ae2 it looks like some actions used the attribute and some used some custom code... how odd...